scripts/index.js

```diff
@@ -15,6 +15,11 @@ if (appConfig.features.useTansaProofing) {
     require('apps/tansa');
 }
 
+// load locale for angular
+if (appConfig.ddangular_i18n) {
```

Comment:
sorry, I was doing some tests, it's supposed to be angular_i18n because it only work for some angular bootstrap components